.coveragerc

```diff
@@ -605,10 +606,12 @@ omit =
     homeassistant/components/switch/rpi_rf.py
     homeassistant/components/switch/snmp.py
     homeassistant/components/switch/switchbot.py
+    homeassistant/components/switch/switcher_kis.py
```

Comment:
Delete

Reply:
Thank you.
Fixed in [this](https://github.com/home-assistant/home-assistant/pull/20830/commits/6d478ac3f47207b344a670fc3a04f1811c544a83) commit.